src/main/java/edu/cornell/kfs/module/cg/document/CuAwardMaintainableImpl.java

```diff
@@ -2,40 +2,13 @@
 
 import org.apache.commons.collections4.CollectionUtils;
 import org.kuali.kfs.krad.util.ObjectUtils;
-import org.kuali.kfs.module.cg.businessobject.Award;
 import org.kuali.kfs.module.cg.businessobject.AwardAccount;
-import org.kuali.kfs.module.cg.businessobject.Proposal;
 import org.kuali.kfs.module.cg.document.AwardMaintainableImpl;
-import org.kuali.kfs.sys.KFSPropertyConstants;
 
 import edu.cornell.kfs.module.cg.businessobject.AwardAccountExtendedAttribute;
 
 public class CuAwardMaintainableImpl extends AwardMaintainableImpl {
 
-    /**
-     * Load related objects from the database as needed.
-     *
-     * @param refreshFromLookup
-     */
-    @Override
-    protected void refreshAward(final boolean refreshFromLookup) {       
-        final Award award = getAward();
-        final Proposal tempProposal = getAward().getProposal();
```

Comment:
it looks like these two proposal steps are the only things not in the base code version of this function.  Should we do these two steps, then call the the super. refreshAward()?
I am not sure what these lines would accomplish, so I am OK moving forward with the total removal.  However, I wanted to ask and think about it.

Reply:
Our old code was retrieving the Award, accessing its Proposal reference, and reassigning the Award's Proposal reference with that same value. The getter and setter for that Proposal reference don't contain extra processing, so our custom handling was effectively leaving the Proposal reference unchanged, rather than refreshing the reference like in base code. I'm not certain why it was set up this way, unless it was modeled after a similar customization elsewhere in our code (but didn't get adjusted properly when it was added to this class).

Given the above, I don't think we need to keep our override of this method.